.github/workflows/typos.yml

```diff
@@ -0,0 +1,57 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+---
+name: Typos
+
+on:
+    workflow_call:
```

Comment:
Is this let empty for future use?

Reply:
Not empty, can just remove the colon after.
This `workflow_call` needs to be specified in order for this workflow to be "reusable"

Reply:
ok!